### PR TITLE
Country picker component

### PR DESCRIPTION
**Pull request by jurczewski**
* Country picker list component
* Download all available countries
* Country Icon component
* Stylization of icon and country picker

### CODE REVIEW

File: pandemania/src/api/FetchAllCountries.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const apiUrl = 'https://api.covid19api.com/countries';
+
+const allCountries = async () => {
+	try {
+		const response = await fetch(apiUrl);
```
Comment (Pawci0) [inline pandemania/src/api/FetchAllCountries.js]: I'd rather use .then() nstead of using await. That way you can be more confident the response is valid

File: pandemania/src/components/CountryPicker/CountryPicker.jsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+		type: 'dark',
+	},
+});
+
+const CountryPicker = () => {
+	const [countries, setCountries] = useState([]);
+
+	const fetchData = async () => {
+		const response = await allCountries();
+		const fetchedCountries = response.map((c) => c);
```
Comment (Pawci0) [inline pandemania/src/components/CountryPicker/CountryPicker.jsx:23]: same thing here

**Review by Pawci0 — CHANGES_REQUESTED** [review]

**Review by Pawci0 — APPROVED** [review]

File: pandemania/src/api/FetchAllCountries.js (hunk 1 above)
Comment (jurczewski) [reply pandemania/src/api/FetchAllCountries.js]: Done 👍

File: pandemania/src/components/CountryPicker/CountryPicker.jsx (hunk 2 above)
Comment (jurczewski) [reply pandemania/src/components/CountryPicker/CountryPicker.jsx:23]: Done 👍

**Pull request merged**